test/visibility/submodule-in-file/errors/MultipleModules.chpl

```diff
@@ -0,0 +1,9 @@
+// This case actually compiles as long as the
+// module doing the including is named appropriately.
+module MultipleModules {
+  include module SubModule;
+  proc main() {
+  }
+}
+module OtherModule {
+}
```

Comment:
Just to make sure I'm not missing anything:  The reason the .good for this one is empty is because nothing is making use of the sub-module, is that correct?  But if we did, it would print out its initialization?  I'd probably change this to do so to lock that in (i.e., having multiple modules, one of which has an include doesn't prevent the submodule from working).

Reply:
(Oh, I'd also move this out of the errors directory?)

Reply:
This case was originally proposed to be an error but when I got to implementing it didn't seem like a big deal. I'll move it out of the errors directory and add some code using OtherModule.